src/include/cell_inline.h

```diff
@@ -1226,6 +1226,26 @@ __wt_cell_unpack_kv(WT_SESSION_IMPL *session, const WT_PAGE_HEADER *dsk, WT_CELL
     __cell_unpack_window_cleanup(session, dsk, NULL, unpack_value);
 }
 
+/*
+ * __wt_cell_get_ta --
+ *     Set the time aggregate from an unpacked address cell.
```

Comment:
How about the comment something like `Get the underlying time aggregate from ...`. The same for the next function.

Reply:
Fixed in 3ddd3c4e38653a29b7110261c7695b473975cf75